src/views/Events/index.jsx

```diff
@@ -141,34 +167,175 @@ const Events = () => {
     return (
       <Grid container justifyContent="center" spacing={6}>
         <Grid item xs={12} lg={10} xl={8}>
-          <Card style={{ padding: '0 3vw' }}>
-            <CardContent>
-              <Grid container direction="row" alignItems="center">
-                <Grid item xs={4} />
-                <Grid item xs={4} align="center">
-                  <CardHeader title={searchPageTitle} />
+          <Card style={{ padding: '0 0vw' }}>
+            <CardHeader
+              title={searchPageTitle}
+              style={{
+                backgroundColor: gordonColors.primary.blue,
+                color: gordonColors.neutral.grayShades[50],
+              }}
+            />
+            <Card style={{ padding: '0 3vw' }}>
```

Comment:
Is this nested card needed?

Reply:
I only ask because it seems odd to have a Card with a header, and then instead of content have another card which has content,

Reply:
If I remember right, I did this this way to make sure the cards stayed separate. There may be a better way to accomplish the same thing-I didn't experiment much with it.